workflow-job

```diff
@@ -330,11 +376,15 @@ flows["${sha1}"] = {
                 def dUser = dockerUser();
                 def dPassword = dockerPassword();
                 sh "docker login -u ${dUser} -p ${dPassword}"
-                sh "docker tag -f ${images.james} linagora/james-project"
+                sh "docker tag -f ${images.jamesCassandra} linagora/james-project"
                 sh "docker push linagora/james-project"
 
                 sh "docker tag -f ${images.jamesJava6} linagora/james-jpa-spring"
                 sh "docker push linagora/james-jpa-spring"
+
+                sh "docker tag -f ${images.jamesJPA} linagora/james-jpa-guice"
+                sh "docker push linagora/james-guice-jpa"
+
```

Comment:
I tried and took more time than estimation and I cannot make them all in parallel, I just make 2 of them can be parallel. I will take a look at more but on another ticket.
I committed the current success case.